package.json

```diff
@@ -284,7 +288,7 @@
     "logError": true
   },
   "lint-staged": {
-    "*.{js,ts,tsx}": [
+    "*.{js,tsx}": [
```

Comment:
prettier is not playing well with `.d.ts` files. I will look further into this, but I wanted to push this for review while I did that

Reply:
for posterity, i believe this is an issue with `prettier-standard`, see https://github.com/sheerun/prettier-standard/issues/111#issuecomment-782924961. probably not worth trying to fix right now since we're only creating declaration files, but once we do a larger scale migration to TS, we should hopefully have this fixed :)